stages/03_build.R

```diff
@@ -14,8 +14,8 @@ save_parquet <- function(file) {
 	file_ext = file_ext(file)
 	make_parquet = c("txt", "bed", "gff")
 	is_README = grepl(file, pattern="README")
-
-	if(file_ext %in% make_parquet & !is_README) {
+	ts_directories <- grepl("^targetscan_7.*", dirname(file), ignore.case = TRUE)
```

Comment:
I would prefer this skipping done during download and using the specific name of the file:

So:

```r
grepl( r'(^(targetscan_70\.zip|targetscan_70_BL_PCT\.zip|TargetScan7_context_scores\.zip)$)', ... )
```

See [raw strings](https://r4ds.hadley.nz/strings.html#sec-raw-strings) for that syntax.

Reply:
I'll go ahead and remove this line.